scene/3d/spring_arm_3d.cpp

```diff
@@ -187,6 +192,7 @@ void SpringArm3D::process_spring() {
 
 		get_world_3d()->get_direct_space_state()->cast_motion(shape_params, motion_delta, motion_delta_unsafe);
 	}
+	collided = motion_delta < 1.0;
```

Comment:
```suggestion

	collided = motion_delta < 1.0;
```
Keep the empty line, just remove the trailing whitespace 🙂

Reply:
S-should be good now ><